projects/docs/docs/manifests/dependencies.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Dependencies.swift
+# description: 'This page documents the models that users can use to define their project: how to initialize them, attributes and their meaning, protocol comformances.'
+slug: '/manifests/dependencies'
+---
+
+:::warning Work in progress
+This feature is currently being worked on and is not ready to be used yet.
+:::
+
+Learn how to get started with `Dependencies.swift` [here](/guides/third-party-dependencies/).
+
+```swift
+import ProjectDescription
+
+let dependencies = Dependencies(
+    carthage: .carthage(
+        [
+            .github(path: "Alamofire/Alamofire", requirement: .exact("5.0.4"))
+        ],
+        options: [.useXCFrameworks, .noUseBinaries]
+    ),
+    swiftPackageManager: nil, // work in progress, pass `nil`
+    platforms: [.iOS]
+)
+```
+
+### Dependencies
+
+A `Dependencies` manifest allows for defining external dependencies for Tuist.
+
+| Property              | Description                                                                      | Type                                                                     | Required | Default                  |
+| --------------------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------ | -------- | ------------------------ |
+| `carthage`            | The description of dependency that can be installed using Carthage.              | [`CarthageDependencies`](#carthage-dependencies)                         | No       | `nil`                    |
+| `swiftPackageManager` | The description of dependency that can be installed using Swift Package Manager. | [`SwiftPackageManagerDependencies`](#swift-package-manager-dependencies) | No       | `nil`                    |
```

Comment:
```suggestion
| `swiftPackageManager` | The description of a dependency that can be installed using Swift Package Manager. | [`SwiftPackageManagerDependencies`](#swift-package-manager-dependencies) | No       | `nil`                    |
```

Reply:
I'm fixing this typo in /ProjectDescription/Dependencies/Dependencies.swift too.